pkg/txn/storage/tae/log.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Matrix Origin
```

Comment:
You are not going to introduce another log, use logutil.   Talk to XZX if you need help.

Reply:
ok